index.html

```diff
@@ -0,0 +1,311 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Dashboard</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <div class="grid">
+        <!--Left SideBar-->
+        <div class="left">
+            <!--Company Logo -->
+            <div class="logoObject">
```

Comment:
use lower case separated with hyphen for naming class. 
Ex: `logo-container`

Reply:
Code classes changed from Camel Case to Kebab Case. Resolved.